### PR TITLE
changes to SQL code to make it work for TSQL

### PR DESCRIPTION
**Pull request by chaerinlee1**
Small syntax changes in the `customers.sql` and `orders.sql` file to make it work for TSQL users.

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ clean-targets:
 
 models:
   jaffle_shop:
+      +as_columnstore: False
```
Comment (dataders) [inline dbt_project.yml]: @chaerinlee1 this isn't relevant for non dbt-sqlserver users
```suggestion
```

**Review by dataders — APPROVED** [review]
@chaerinlee1 this looks good to me! all of these syntax tweaks should run cross platform

File: models/orders.sql [hunk 2]
```diff
@@ -25,29 +25,29 @@ order_payments as (
...
 
     select
-        orders.order_id,
-        orders.customer_id,
-        orders.order_date,
-        orders.status,
+        o.order_id,
+        o.customer_id,
+        o.order_date,
+        o.status,
```
Comment (dataders) [inline models/orders.sql]:
```suggestion
        orders.order_id,
        orders.customer_id,
        orders.order_date,
        orders.status,
```

File: models/orders.sql [hunk 3]
```diff
@@ -25,29 +25,29 @@ order_payments as (
...
 
         {% for payment_method in payment_methods -%}
 
-        order_payments.{{ payment_method }}_amount,
+        op.{{ payment_method }}_amount,
```
Comment (dataders) [inline models/orders.sql]:
```suggestion
        order_payments.{{ payment_method }}_amount,
```

File: models/orders.sql [hunk 4]
```diff
@@ -25,29 +25,29 @@ order_payments as (
...
 
         {% endfor -%}
 
-        order_payments.total_amount as amount
+        op.total_amount as amount
```
Comment (dataders) [inline models/orders.sql]:
```suggestion
        order_payments.total_amount as amount
```

File: models/orders.sql [hunk 5]
```diff
@@ -25,29 +25,29 @@ order_payments as (
...
 
-    from orders
+    from orders o
```
Comment (dataders) [inline models/orders.sql]:
```suggestion
    from orders
```

File: models/orders.sql [hunk 6]
```diff
@@ -25,29 +25,29 @@ order_payments as (
...
 
-    left join order_payments using (order_id)
+    left join order_payments op on o.order_id=op.order_id
```
Comment (dataders) [inline models/orders.sql]:
```suggestion
    left join order_payments on orders.order_id = order_payments.order_id
```

File: models/customers.sql [hunk 7]
```diff
@@ -26,40 +26,40 @@ customer_orders as (
...
-        customer_orders.most_recent_order,
-        customer_orders.number_of_orders,
-        customer_payments.total_amount as customer_lifetime_value
+        c.customer_id,
+        c.first_name,
+        c.last_name,
+        co.first_order,
+        co.most_recent_order,
+        co.number_of_orders,
+        cp.total_amount as customer_lifetime_value
```
Comment (dataders) [inline models/customers.sql]:
```suggestion
        customers.customer_id,
        customers.first_name,
        customers.last_name,
        customer_orders.first_order,
        customer_orders.most_recent_order,
        customer_orders.number_of_orders,
        customer_payments.total_amount as customer_lifetime_value
```

File: models/customers.sql [hunk 8]
```diff
@@ -26,40 +26,40 @@ customer_orders as (
...
 
-    from customers
+    from customers c
```
Comment (dataders) [inline models/customers.sql]:
```suggestion
    from customers
```

File: models/customers.sql [hunk 9]
```diff
@@ -26,40 +26,40 @@ customer_orders as (
...
 
-    left join customer_orders using (customer_id)
+    left join customer_orders co on c.customer_id=co.customer_id
 
-    left join customer_payments using (customer_id)
+    left join customer_payments cp on cp.customer_id=c.customer_id
```
Comment (dataders) [inline models/customers.sql]:
```suggestion
    left join customer_payments
        on  customers.customer_id = customer_payments.customer_id
```

File: models/customers.sql [hunk 10]
```diff
@@ -26,40 +26,40 @@ customer_orders as (
...
+        co.first_order,
+        co.most_recent_order,
+        co.number_of_orders,
+        cp.total_amount as customer_lifetime_value
 
-    from customers
+    from customers c
 
-    left join customer_orders using (customer_id)
+    left join customer_orders co on c.customer_id=co.customer_id
```
Comment (dataders) [inline models/customers.sql]:
```suggestion
    left join customer_orders
        on customers.customer_id = customer_orders.customer_id
```

File: models/customers.sql [hunk 11]
```diff
@@ -26,40 +26,40 @@ customer_orders as (
...
     select
-        orders.customer_id,
+        o.customer_id,
         sum(amount) as total_amount
 
-    from payments
+    from payments p
 
-    left join orders using (order_id)
+    left join orders o on o.order_id=p.order_id
```
Comment (dataders) [inline models/customers.sql]:
```suggestion
    left join orders on
         payments.order_id = orders.order_id
```

File: models/customers.sql [hunk 12]
```diff
@@ -26,40 +26,40 @@ customer_orders as (
...
 
 customer_payments as (
 
     select
-        orders.customer_id,
+        o.customer_id,
         sum(amount) as total_amount
 
-    from payments
+    from payments p
```
Comment (dataders) [inline models/customers.sql]:
```suggestion
    from payments
```

File: models/customers.sql [hunk 13]
```diff
@@ -26,40 +26,40 @@ customer_orders as (
...
-    group by 1
+    group by customer_id
 
 ),
 
 customer_payments as (
 
     select
-        orders.customer_id,
+        o.customer_id,
```
Comment (dataders) [inline models/customers.sql]:
```suggestion
        orders.customer_id,
```

File: models/customers.sql [hunk 14]
```diff
@@ -26,40 +26,40 @@ customer_orders as (
...
         sum(amount) as total_amount
 
-    from payments
+    from payments p
 
-    left join orders using (order_id)
+    left join orders o on o.order_id=p.order_id
 
-    group by 1
+    group by o.customer_id
```
Comment (dataders) [inline models/customers.sql]:
```suggestion
    group by orders.customer_id
```

**Review by dataders — CHANGES_REQUESTED** [review]
@chaerinlee1 I'd like to propose as few changes as possible, so removed your table aliasing (e.g. `o.` and `op.`)

**Review by dataders — APPROVED** [review]
thanks @chaerinlee1

File: models/orders.sql [hunk 15]
```diff
@@ -47,7 +47,7 @@ final as (
 
     from orders
 
-    left join order_payments using (order_id)
+    left join order_payments on orders.order_id = order_payments.order_id
```
Comment (annafil) [inline models/orders.sql:50]:
```suggestion
    left join order_payments 
        on orders.order_id = order_payments.order_id
```

**Review by annafil — APPROVED** [review]
Thank you both for these changes! Agree with Anders re: removing table aliases, this is consistent with the [style guide](https://github.com/dbt-labs/corp/blob/master/dbt_style_guide.md) we use in our projects.

**Pull request merged**